loopchain/blockchain/blockchain.py

```diff
@@ -65,10 +65,7 @@ class BlockChain:
     PREPS_KEY = b'preps_key'
     INVOKE_RESULT_BLOCK_HEIGHT_KEY = b'invoke_result_block_height_key'
 
-    def __init__(self, channel_name=None, store_id=None, block_manager=None):
-        if channel_name is None:
-            channel_name = conf.LOOPCHAIN_DEFAULT_CHANNEL
-
+    def __init__(self, channel_name, store_id=None, block_manager=None):
```

Comment:
Can you remove the default value of `store_id` (required param)?
When I glanced first here, I thought DB name can be `.../db_`.

But I figured out that the variable always be a valid name from what it be initialized.

Reply:
done